src/renderer/modules/components/KeybindItem.tsx

```diff
@@ -0,0 +1,295 @@
+            style={{ width: "24px", height: "24px" }}>
+            <path d="M12 2c5.523 0 10 4.477 10 10s-4.477 10-10 10S2 17.523 2 12 6.477 2 12 2Zm3.53 6.47-.084-.073a.75.75 0 0 0-.882-.007l-.094.08L12 10.939l-2.47-2.47-.084-.072a.75.75 0 0 0-.882-.007l-.094.08-.073.084a.75.75 0 0 0-.007.882l.08.094L10.939 12l-2.47 2.47-.072.084a.75.75 0 0 0-.007.882l.08.094.084.073a.75.75 0 0 0 .882.007l.094-.08L12 13.061l2.47 2.47.084.072a.75.75 0 0 0 .882.007l.094-.08.073-.084a.75.75 0 0 0 .007-.882l-.08-.094L13.061 12l2.47-2.47.072-.084a.75.75 0 0 0 .007-.882l-.08-.094-.084-.073.084.073Z" />
+          </svg>
+          <Text.Normal className="button-text">Stop Recording</Text.Normal>
+        </div>
+      </div>
+    );
+  }
+
+  return (
+    <div className="buttons">
+      <div
+        className={`button ${props.isRecording ? "recording" : ""}`}
+        onClick={(event) => {
+          event.stopPropagation();
+          props.toggleRecording();
+        }}>
+        <svg
+          className="icon"
+          fill={props.isRecording ? "#ff0000" : "var(--channel-icon)"}
+          viewBox="0 0 24 24"
+          style={{ width: "24px", height: "24px" }}>
+          <path d="M19.745 5a2.25 2.25 0 0 1 2.25 2.25v9.505a2.25 2.25 0 0 1-2.25 2.25H4.25A2.25 2.25 0 0 1 2 16.755V7.25A2.25 2.25 0 0 1 4.25 5h15.495Zm-2.495 9.5H6.75l-.102.007a.75.75 0 0 0 0 1.486L6.75 16h10.5l.102-.007a.75.75 0 0 0 0-1.486l-.102-.007ZM16.5 11a1 1 0 1 0 0 2 1 1 0 0 0 0-2Zm-2.995 0a1 1 0 1 0 0 2 1 1 0 0 0 0-2Zm-3 0a1 1 0 1 0 0 2 1 1 0 0 0 0-2Zm-3 0a1 1 0 1 0 0 2 1 1 0 0 0 0-2ZM6 8a1 1 0 1 0 0 2 1 1 0 0 0 0-2Zm2.995 0a1 1 0 1 0 0 2 1 1 0 0 0 0-2Zm3 0a1 1 0 1 0 0 2 1 1 0 0 0 0-2Zm3 0a1 1 0 1 0 0 2 1 1 0 0 0 0-2Zm3 0a1 1 0 1 0 0 2 1 1 0 0 0 0-2Z" />
+        </svg>
+        <Text.Normal className="button-text">Edit Keybind</Text.Normal>
+      </div>
+      <div
+        className={`button ${props.isRecording ? "recording" : ""}`}
+        onClick={(event) => {
+          event.stopPropagation();
+          props.clearKeybind();
+        }}>
+        <svg
+          className="icon"
+          fill={props.isRecording ? "#ff0000" : "var(--channel-icon)"}
+          viewBox="0 0 24 24"
+          style={{ width: "24px", height: "24px" }}>
+          <path d="M12 2c5.523 0 10 4.477 10 10s-4.477 10-10 10S2 17.523 2 12 6.477 2 12 2Zm3.53 6.47-.084-.073a.75.75 0 0 0-.882-.007l-.094.08L12 10.939l-2.47-2.47-.084-.072a.75.75 0 0 0-.882-.007l-.094.08-.073.084a.75.75 0 0 0-.007.882l.08.094L10.939 12l-2.47 2.47-.072.084a.75.75 0 0 0-.007.882l.08.094.084.073a.75.75 0 0 0 .882.007l.094-.08L12 13.061l2.47 2.47.084.072a.75.75 0 0 0 .882.007l.094-.08.073-.084a.75.75 0 0 0 .007-.882l-.08-.094L13.061 12l2.47-2.47.072-.084a.75.75 0 0 0 .007-.882l-.08-.094-.084-.073.084.073Z" />
+        </svg>
+        <Text.Normal className="button-text">Clear Keybind</Text.Normal>
+      </div>
+    </div>
+  );
+};
+
+export default class KeybindRecorder extends React.Component<KeybindProps, KeybindState> {
```

Comment:
You have to export two components from this file. One is KeybindRecorder, the clean component without any of our modifications, which must be as similar as possible to the original. The other is KeybindRecorderItem, must be a KeybindRecorder wrapped in a FormItem.